src/simulators/morse_ros/morse_ros/scripts/local_collision.py

```diff
@@ -53,6 +55,12 @@ def __init__(self, config = None, rate = 1, debug = False):
         rospy.sleep(2.0)
         self.rate = rospy.Rate(rate)
 
+    def reset_humans(self, num_hum):  #Write this function
```

Comment:
This function is available at #3

Reply:
@vbalaji21 yes bro, I should have used the reset method. Currently, I don't reset the humans list but I alter the list index based on number of required humans. I am currently reusing the methods in #3 and brought it into this branch